tests/test_custom_type_setup.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+import gc
+import weakref
+
+import pytest
+
+import env  # noqa: F401
+from pybind11_tests import custom_type_setup as m
+
+
+@pytest.fixture
+def gc_tester():
+    """Tests that an object is garbage collected."""
+
+    weak_refs = []
+
+    def add_ref(obj):
+        # PyPy does not support `gc.is_tracked`.
```

Comment:
Is this still needed? (PyPy seems to be skipped entirely.)

Reply:
I removed PyPy because it is unreliable but it still seemed somewhat useful to retain those workarounds in case someone wants to make the test more reliable in the future.